src/Lifecycle.js

```diff
@@ -323,16 +323,15 @@ function _handleLoadSessionFailure(e) {
         error: e.message,
     });
 
-    return modal.finished.then(([success]) => {
-        if (success) {
-            // user clicked continue.
-            _clearStorage();
-            return false;
-        }
+    const [success] = await modal.finished;
+    if (success) {
+        // user clicked continue.
+        await _clearStorage();
```

Comment:
is this fixing any issue that we were having? or just good to do?

Reply:
It prevents a possible issue when a session gets invalidated and someone logging back in whilst the storage clearing is still working but no known reports of it